cellprofiler/modules/relateobjects.py

```diff
@@ -1,6 +1,16 @@
 # coding=utf-8
 
-"""
+import numpy as np
+
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
+import _help
```

Comment:
It looks like only this import (`import _help`) is necessary. You can also move all the imports from lines 65-73 up here, too:

```python
import re

import numpy
import scipy.ndimage
import skimage.segmentation

import cellprofiler.measurement
import cellprofiler.module
import cellprofiler.setting
```